cli/tests/pcluster/aws/test_ec2.py

```diff
@@ -524,7 +524,7 @@ def get_describe_capacity_reservation_mocked_request(capacity_reservations, stat
                 for capacity_reservation in capacity_reservations
             ]
         },
-        expected_params={"CapacityReservationIds": capacity_reservations},
+        expected_params={"CapacityReservationIds": capacity_reservations},  # TODO add: "ReservationType": None},
```

Comment:
missing TODO

Reply:
removed. This is not implemented on ec2 API.